desktop-app/index.html

```diff
@@ -6,16 +6,16 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
   <title>Tinder Desktop</title>
   <meta name="description" content="A cross-platform desktop Tinder client">
-
+  
   <link rel="stylesheet" href="css/normalize.min.css"/>
   <link href='https://fonts.googleapis.com/css?family=Raleway' rel='stylesheet' type='text/css'>
   <link href='https://fonts.googleapis.com/css?family=Merriweather' rel='stylesheet' type='text/css'>
   <link rel="stylesheet" href="css/font-awesome.min.css">
   <link rel="stylesheet" href="css/app.css"/>
   <link rel="stylesheet" href="css/sweet-alert.css"/>
-
+  
+  <script src="https://twemoji.maxcdn.com/2/twemoji.min.js"></script>
```

Comment:
There's also privacy implications, maybe we want to ship it locally and avoid outside dependencies?